docs/BreakingChanges/028 MachineKey.Encode and MachineKey.Decode methods.md

```diff
@@ -0,0 +1,23 @@
+## 28: MachineKey.Encode and MachineKey.Decodemethods
```

Comment:
{
  "Id": "28",
  "Title": "MachineKey.Encode and MachineKey.Decodemethods",
  "Details": "These methods are now obsolete. Compilation of code that calls these methods produces a compiler warning.",
  "Suggestion": "The recommended alternatives are [MachineKey.Protect](https://msdn.microsoft.com/en-us/library/system.web.security.machinekey.protect(v=vs.110).aspx) and [MachineKey.Unprotect](https://msdn.microsoft.com/en-us/library/system.web.security.machinekey.unprotect(v=vs.110).aspx). Alternatively, the build warnings can be suppressed or they can be avoided by using an older compiler. The APIs are still supported.",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#asp",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Web.Security.MachineKey.Encode(System.Byte[],System.Web.Security.MachineKeyProtection)",
    "M:System.Web.Security.MachineKey.Decode(System.String,System.Web.Security.MachineKeyProtection)"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": true,
  "ImpactScope": 2
}

**Parsed APIs:**
- [x] M:System.Web.Security.MachineKey.Encode(System.Byte[],System.Web.Security.MachineKeyProtection)
- [x] M:System.Web.Security.MachineKey.Decode(System.String,System.Web.Security.MachineKeyProtection)